bundles/go/go_completer.moon

```diff
@@ -0,0 +1,50 @@
+import app, config, interact from howl
+import ProcessBuffer from howl.ui
+import Process from howl.io
+import Matcher from howl.util
+append = table.insert
+
+install_gocode = ->
+  p = Process {
+    cmd: { 'go', 'get', '-u', '-v', 'github.com/nsf/gocode' },
+    read_stdout: true,
+    read_stderr: true,
+  }
+
+  buffer = ProcessBuffer p
+  editor = app\add_buffer buffer
+  editor.cursor\eof!
+  buffer\pump!
+  
+run_gocode = (context) ->
+  cmd = { 'gocode', '-f=csv', 'autocomplete', context.pos-1 }
```

Comment:
So it's been a while since I last had a look at Go, but can we actually rely on `gocode` being available in `$PATH`? It will be installed into `$GOPATH/bin` by the install command, so perhaps that's what we should try to run instead?